schedule/reschedule.py

```diff
@@ -364,6 +367,21 @@ def reschedule_card(cid, fsrs: FSRS, recompute=False):
         fsrs.set_card(card)
         fsrs.set_fuzz_factor(cid, card.reps)
         new_ivl = fsrs.next_interval(s)
+
+        dr = fsrs.desired_retention
+        odds = dr / (1 - dr)
+
+        reduced_odds = (1 - fsrs.reschedule_threshold) * odds
+        fsrs.desired_retention = reduced_odds / (reduced_odds + 1)
+        adjusted_ivl_upper = fsrs.next_interval(s)
```

Comment:
Is fuzz applied when calculating `adjusted_ivl_upper` and `adjusted_ivl_lower`? Ideally, fuzz should not be applied.

I am asking this because apparently there are two different functions with the same name.

https://github.com/open-spaced-repetition/fsrs4anki-helper/blob/8c183e9fb53c4812831914bd8ba84905823fddfd/schedule/reschedule.py#L163-L165